common/containers/Tabs/Contracts/components/Interact/components/InteractExplorer/index.tsx

```diff
@@ -137,14 +159,18 @@ class InteractExplorerClass extends Component<Props, State> {
 
   private contractOptions = () => {
     const { contractFunctions } = this.props;
-
-    return Object.keys(contractFunctions).map(name => {
-      return (
-        <option key={name} value={name}>
-          {name}
-        </option>
-      );
+    let transformedContractFunctions: ContractOption[] = [];
+    Object.keys(contractFunctions).forEach(contractFunction => {
+      const contract = contractFunctions[contractFunction];
+      if (contract) {
```

Comment:
Is this a typing issue? I think you can type `contractFunction: keyof contractFunctions`.

Reply:
Also nitpick, you could just make `transformedContractFunctions` a map of Object.keys instead of concatting the array each time.

Reply:
👍